### PR TITLE
[hi] Localize  content/hi/docs/reference/glossary/container.md

### PR DESCRIPTION
**Pull request by amolmote**
Fixes #35600

### CODE REVIEW

File: content/hi/docs/reference/glossary/container.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: कंटेनर (Container)
+id: container
+date: 2018-04-12
+full_link: /docs/concepts/containers/
+short_description: >
+   एक हल्की और पोर्टेबल एक्सेक्यूटबल योग्य इमेज जिसमें सॉफ़्टवेयर और उसकी सभी निर्भरताएँ शामिल हैं।
```
Comment (Garima-Negi) [inline content/hi/docs/reference/glossary/container.md:7]: एक्सेक्यूटबल योग्य = निष्पाद्य

File: content/hi/docs/reference/glossary/container.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+aka: 
+tags:
+- fundamental
+- workload
+---
+ एक हल्की और पोर्टेबल एक्सेक्यूटबल योग्य इमेज जिसमें सॉफ़्टवेयर और उसकी सभी निर्भरताएँ शामिल हैं।
```
Comment (Garima-Negi) [inline content/hi/docs/reference/glossary/container.md:14]: same comment as above

File: content/hi/docs/reference/glossary/container.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+<!--more--> 
+
+कंटेनर अलग-अलग क्लाउड या ओएस(OS) एनवायरनमेन्ट्स में और आसान स्केलिंग के लिए डिप्लॉयमेंट को आसान बनाने के लिए, अंतर्निहित होस्ट इन्फ्रास्ट्रक्चर से अनुप्रयोगों को अलग करते हैं।
+
```
Comment (Garima-Negi) [inline content/hi/docs/reference/glossary/container.md:19]: This PR is missing sentences present in `en` content. Please check : [https://github.com/kubernetes/website/blob/main/content/en/docs/reference/glossary/container.md](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/glossary/container.md)

**Review by Garima-Negi — CHANGES_REQUESTED** [review]

File: content/hi/docs/reference/glossary/container.md (hunk 1 above)
Comment (amolmote) [reply content/hi/docs/reference/glossary/container.md:7]: thanks @Garima-Negi  for suggestions, I will update it asap.

**Comment by divya-mohan0209** [comment]
Hello @amolmote , just following up. Do you still have the cycles to work on this?

**Comment by divya-mohan0209** [comment]
Additionally, please synch your local branch to the current main branch before pushing any further commits.

**Comment by amolmote** [comment]
Yes i need to do some changes in this pr will do it today and sync the main branch

**Comment by divya-mohan0209** [comment]
Hello @amolmote , I understand you're busy & that's okay. Thank you for your efforts on this PR. However, since there has been no action from ~5 months, I will be closing it off. Whenever you have the bandwidth to address feedback, please feel free to reopen & we can resume where we left off.
/close

**Pull request closed without merging**